packages/tocco-util/src/notifier/components/TitleMessage.js

```diff
@@ -1,40 +1,67 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import {FormattedMessage} from 'react-intl'
-import {FormattedValue} from 'tocco-ui'
+import {FormattedValue, Typography} from 'tocco-ui'
 import _isString from 'lodash/isString'
 
+const containsHtml = s => s && /<\/?[a-zA-Z0-9]*\/?>/.test(s)
 const isKey = s => s && s.startsWith('client.')
 
-const Content = ({content}) =>
-  _isString(content)
-    ? (isKey(content) ? <FormattedMessage id={content}/> : <FormattedValue type="html" value={content}/>)
-    : content
+const Content = props => {
+  const {content} = props
+  let output = content
+
+  if (_isString(content)) {
+    if (containsHtml(content)) {
+      output = <FormattedValue type="html" value={content}/>
+    } else {
+      output = isKey(content) ? <FormattedMessage id={content}/> : content
+      output = <props.tag>{output}</props.tag>
+    }
+  }
+  return output
```

Comment:
as mentioned in another PR please do rather create a new const than reassign a let var and give proper names.